ISISComputingGroup/ibex_gui
# Ticket 5245: Run OPI checker in CI

### Description of work

See https://github.com/ISISComputingGroup/IBEX/issues/5245

Includes changes from https://github.com/ISISComputingGroup/IBEX/issues/4891 so it can run properly

To test:
* Confirm that a subset of the OPI checker scripts are being run as part of the jenkins build
* These tests currently pass

---

#### Code Review


### Final Steps
